geocoder-data-provider/caddy-minio-pvc-template.yaml

```diff
@@ -0,0 +1,382 @@
+    #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+    # TODO
+    # Questions:
+    #    1. why is the route not listed in the overview page?  trivial but it bugs me. it WAS showing up before!
+    #    2. Should routes be internal only? e.g. route-to-this-app.pathfinder.bcgov
+    #
+  spec:
+    #host: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}${DEPLOYMENT_ENV}.${ROUTE_SUBDOMAIN}
+    host: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}.${ROUTE_SUBDOMAIN}
+    port:
+      targetPort: 9000-tcp
+    tls:
+      termination: edge
+    to:
+      kind: Service
+      name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+      #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+      weight: 100
+    wildcardPolicy: None
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}
+    #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+    # TODO
+    # Questions:
+    #    1. why is the route not listed in the overview page?  trivial but it bugs me. it WAS showing up before!
+    #    2. Should routes be internal only? e.g. route-to-this-app.pathfinder.bcgov
+    #
+  spec:
+    # host: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}.${ROUTE_SUBDOMAIN}
+    #host: ${APPLICATION_NAME}${DEPLOYMENT_ENV}.${ROUTE_SUBDOMAIN}
+    host: ${APPLICATION_NAME}.${ROUTE_SUBDOMAIN}
+    port:
+      targetPort: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}
+    tls:
+      termination: edge
+    to:
+      kind: Service
+      name: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}
+      #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+      weight: 100
+    wildcardPolicy: None
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      env: internal
+      route: within-cluster
+    name: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}-int
+    #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+    # TODO
+    # Questions:
+    #    1. why is the route not listed in the overview page?  trivial but it bugs me. it WAS showing up before!
+    #    2. Should routes be internal only? e.g. route-to-this-app.pathfinder.bcgov
+    #
+  spec:
+    host: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}.${INTERNAL_ROUTE_SUBDOMAIN}
+    port:
+      # targetPort: 8080-tcp
+      targetPort: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}
+    to:
+      kind: Service
+      name: ${CADDY_SERVICE_NAME}-${APPLICATION_NAME}
+      #name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+      weight: 100
+    wildcardPolicy: None
+# TODO
+# What to name This
+# What size works?
+#
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    annotations:
+      #volume.beta.kubernetes.io/storage-class: gluster-file
+      volume.beta.kubernetes.io/storage-provisioner: netapp.io/trident
+    finalizers:
+    - kubernetes.io/pvc-protection
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}-pv
+  spec:
+    accessModes:
+    - ReadWriteOnce
```

Comment:
seems this need to be ReadWriteMany, as both minio and caddy are trying to access the pvc

Reply:
i think I had to do that at one point.